src/app/dataset-view/additional-components/dataset-settings-component/tabs/webhooks/dataset-settings-webhooks-tab.component.html

```diff
@@ -0,0 +1,113 @@
+                <!-- Event Column -->
+                <ng-container matColumnDef="event">
+                    <th mat-header-cell *matHeaderCellDef>Event</th>
+                    <td mat-cell *matCellDef="let element" style="width: 60%">
+                        <div class="d-flex flex-column justify-content-center py-3 table-cell-height">
+                            <p class="fw-semibold">{{ element.label }}</p>
+                            <a
+                                class="mt-1 truncate-target-url"
+                                [href]="element.targetUrl"
+                                [title]="element.targetUrl"
+                                target="_blank"
+                            >
+                                {{ element.targetUrl }}
+                            </a>
+                        </div>
+                    </td>
+                </ng-container>
+
+                <!-- Status Column -->
+                <ng-container matColumnDef="status">
+                    <th mat-header-cell class="table-header-background" *matHeaderCellDef>Status</th>
+                    <td mat-cell *matCellDef="let element">
+                        <div
+                            [class]="
+                                'd-flex align-items-center status ' +
+                                webhookStatusBadgeOptions(element.status).className
+                            "
+                        >
+                            <mat-icon class="mat-icon-size">{{
+                                webhookStatusBadgeOptions(element.status).iconName
+                            }}</mat-icon>
+                            <span>
+                                {{ element.status }}
+                            </span>
+                        </div>
+                    </td>
+                </ng-container>
+
+                <!-- Actions Column -->
+                <ng-container matColumnDef="actions">
+                    <th mat-header-cell *matHeaderCellDef>Actions</th>
+                    <td mat-cell *matCellDef="let element">
+                        <div class="d-flex flex-row gap-2 py-3">
+                            <mat-icon (click)="viewWebhook()" class="mat-icon-action-size" [title]="'monitoring'"
+                                >visibility</mat-icon
+                            >
+                            <mat-icon (click)="editWebhook(element)" class="mat-icon-action-size" [title]="'edit'"
+                                >edit</mat-icon
+                            >
+                            <mat-icon
+                                class="mat-icon-action-size"
+                                (click)="resumeWebhook(element.id)"
+                                *ngIf="
+                                    element.status === WebhookSubscriptionStatus.Paused &&
+                                    element.status !== WebhookSubscriptionStatus.Unverified
```

Comment:
Can never be paused an unverified at the same time. The 2nd condition is not needed

Reply:
fixed